bin/adam-shell

```diff
@@ -22,6 +22,12 @@ ADAM_REPO="$(cd `dirname $0`/..; pwd)"
 
 ADAM_JARS=$("$ADAM_REPO"/bin/compute-adam-jars.sh)
 
+# does the user have ADAM_OPTS set? if yes, then warn
+if [[ -z $@ &&  ! -z "$ADAM_OPTS" ]]; then
```

Comment:
`s/ ! -z/-n/` here and below? (nit: note the removal of the extra space as well)